### PR TITLE
Feature/blog app views

### PR DESCRIPTION
**Pull request by carlosmuerto**
### Project requirements
- Implement the design from the sneak peek wireframes.
    - Styling is nice to have but focus on making sure everything functions first. (Note: Ignore login logic, you would be using Devise in coming milestone)

- Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).

### `./Item__creation_comands.md` has item creation commands

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1,2 +1,9 @@
-<h1>Users#index</h1>
```
Comment (SajjadAhmad14) [inline app/views/users/index.html.erb:1]: Please follow [this](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_root_users.png?token=GHSAT0AAAAAAB34BLWTYJJHKCGPEGXPRQG4Y4JF7UA) wireframe for user's index page. The layout of the page should be the same as in the given wireframe.

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1,2 +1,16 @@
-<h1>Users#show</h1>
```
Comment (SajjadAhmad14) [inline app/views/users/show.html.erb:1]: Please follow [this](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAAB34BLWS77NKSB5PMIHZJQ3GY4JGA2Q) wireframe for user's show page. The layout of the page should be the same as in the given wirefram.

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1,2 +1,18 @@
-<h1>Posts#index</h1>
```
Comment (SajjadAhmad14) [inline app/views/posts/index.html.erb:1]: Please follow [this](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=GHSAT0AAAAAAB34BLWT5JLXGA5IRSWAYZXAY4JGBTQ) wireframe for posts index page. The layout of the page should be the same as in the given wirefram.

File: app/views/posts/show.html.erb [hunk 4]
```diff
@@ -1,2 +1,24 @@
-<h1>Posts#show</h1>
-<p>Find me in app/views/posts/show.html.erb</p>
```
Comment (SajjadAhmad14) [inline app/views/posts/show.html.erb:2]: Please make the layout of the post show page that displays post with all comments listed as shown on [this](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_single_post.png?token=GHSAT0AAAAAAB34BLWSDE3BKERP263UBM3QY4JGD6A) wireframe

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Carlos, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @carlosmuerto 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**